packages/app/src/containers/LandingPage/index.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import {
+    ImageBackground,
+    Image,
+    Text,
+    View,
+    StyleSheet
+} from 'react-native';
+import { Button } from "react-native-elements";
+
+interface LandingPageState {
+}
+
+interface OwnProps {
+    navigation: any
+}
+
+type Props = OwnProps;
+
+class LandingPage extends React.Component<Props, LandingPageState> {
+    constructor(props: Props) {
+        super(props);
+
+        this.state = {
+        };
+    }
+
+
+    public componentDidMount() {
```

Comment:
do we need this?

Reply:
@petermsouzajr  yes we do not need this so i will just removed